### PR TITLE
[ISSUE #19981] fix unexpected tag error

### PR DESCRIPTION
**Pull request by maxi297**
## What
Fix this issue https://github.com/airbytehq/airbyte/actions/runs/3951883480

### CODE REVIEW

File: .github/workflows/publish-cdk-command-manually.yml [hunk 1]
```diff
@@ -167,7 +166,6 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
       - name: Post success to Slack channel dev-connectors-extensibility
-        if: success()
```
Comment (maxi297) [inline .github/workflows/publish-cdk-command-manually.yml:170]: Based on https://docs.github.com/en/actions/learn-github-actions/expressions#success, this is the default condition so no need to specify it here

File: .github/workflows/publish-cdk-command-manually.yml [hunk 2]
```diff
@@ -52,15 +52,14 @@ jobs:
           new_version="$(grep -i 'current_version = ' .bumpversion.cfg | sed -e 's/.* = //')"
           awk -v NEW_VERSION="$new_version" -v CHANGELOG_MESSAGE="${{ github.event.inputs.changelog-message }}" 'NR==3{print "## " NEW_VERSION "\n" CHANGELOG_MESSAGE "\n"}1' CHANGELOG.md > tmp && mv tmp CHANGELOG.md
       - name: Commit and Push Changes
-        if: success()
```
Comment (maxi297) [inline .github/workflows/publish-cdk-command-manually.yml:55]: Based on https://docs.github.com/en/actions/learn-github-actions/expressions#success, this is the default condition so no need to specify it here

File: .github/workflows/publish-cdk-command-manually.yml [hunk 3]
```diff
@@ -195,7 +193,7 @@ jobs:
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN_AIRBYTE_TEAM }}
       - name: Post failure to Slack channel dev-connectors-extensibility
-        if: !success()
+        if: ${{ failure() }}
```
Comment (erohmensing) [inline .github/workflows/publish-cdk-command-manually.yml:196]: nit: `if: failure()` should work too

File: .github/workflows/publish-cdk-command-manually.yml (hunk 1 above)
Comment (erohmensing) [inline .github/workflows/publish-cdk-command-manually.yml:170]: I think we still want to keep this one - We don't want to post success to slack every case, only if we were successful (Different from the one you removed in `Commit and push changes` , which we want it to attempt to do each time)

**Review by erohmensing — CHANGES_REQUESTED** [review]

**Review by erohmensing — APPROVED** [review]

**Pull request merged**